engine/core/src/main/java/org/datacleaner/lifecycle/AssignConfiguredPropertiesHelper.java

```diff
@@ -28,7 +28,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-final class AssignConfiguredPropertiesHelper {
+public final class AssignConfiguredPropertiesHelper {
```

Comment:
It doesn't matter much, but since I couldn't find any other reference to ```AssignConfiguredPropertiesHelper``` then I guess there is no need to make it public.